### PR TITLE
Task 6: Overall UI design

### PR DESCRIPTION
**Pull request by aka-wtag**
Objective:
- [x] Implement the UI of the app including the previous features.
- [x] The application will start with a splash screen 
- [x] When no task is added blank image will be shown
- [x] When create button is clicked a text area will be shown to add a task
- [x] Tasks are shown in 3 columns
- [x] When task is completed the completion time of the task will be shown 
- [x] A toast message will appear when task is added or edited and when task title is empty

https://github.com/aks-wtag/todo-app/assets/152831915/141793ea-a7f8-425c-a25e-aa2f333b78b5

### CODE REVIEW

File: css/mystrap.css
Comment (mza-wtag) [inline css/mystrap.css:1]: Please provide a meaningful name for the file.

File: css/mystrap.css [hunk 1]
```diff
@@ -0,0 +1,82 @@
...
+    outline: none;
+}
+
+.button,
+.button-icon,
+.button-primary,
+.button-secondary,
+.button-ternary {
+    cursor: pointer;
+    font-weight: 500;
```
Comment (mza-wtag) [inline css/mystrap.css:38]: Take it within a variable, please.

File: js/utility.js [hunk 2]
```diff
@@ -1,11 +1,61 @@
...
+    } else {
+        $messageElement.classList.add("error-message");
+    }
+
+    $messageElement.append($spanElement);
+
+    setTimeout(() => {
+        $messageElement.classList.add("hide");
+    }, 1500);
 };
```
Comment (mza-wtag) [inline js/utility.js:61]: Implement this functionality for adding tasks as well.

**Review by mza-wtag — COMMENTED** [review]
Please update the UI of the cancel button and fix the issues. When you are in edit mode and click the complete button, it should be completed with the updated values, redirect it to the initial completed state, and handle the edit button properly, as after the task is done, the edit button is still remaining.

**Review by ich-wtag — COMMENTED** [review]
Screenshot from 2024-03-07 09-49-45
When you click on this blank box, the blank input field should be opened.


[Screencast from 03-07-2024 09:50:38 AM.webm](https://github.com/aks-wtag/todo-app/assets/153156568/caf72088-2894-471c-a2d2-a90639623c28)
Like when you add any task filter button is not reflecting the changes. I noticed you have different Feature branch for that, if it is please move this buttons **UI** to that branch. It's better to have related functionalities to be in single branch.

Finally you have added a render function. Can you please use this for function for add task too. Cause each function should be tried to keep pure and reusable. So as render function is a reusable function we should reuse it and modify it to use maximum places as much as we can. Thanks bhai.

**Comment by aks-wtag** [comment]
@ich-wtag bhai i have shown the text area when the image is clicked and removed the filter buttons. Bhai the render function is not needed for add, delete or edit task here but while implementing the pagination in the upcoming branch i had to use the render function for add and delete so i have changed it there. Thanks.

**Review by nmi-wtag — CHANGES_REQUESTED** [review]
Screenshot from 2024-03-07 10-55-40

Clicking on search icon is not opening search input box

**Review by nmi-wtag — CHANGES_REQUESTED** [review]
[Screencast from 7-3-24 11:03:29 পূর্বাহ্ণ +06.webm](https://github.com/aks-wtag/todo-app/assets/148762372/db88c7de-bd74-4927-8647-6cd75a4f198f)

Please fix it

**Review by ich-wtag — APPROVED** [review]

**Review by nmi-wtag — APPROVED** [review]

**Review by rka-wtag — APPROVED** [review]

**Review by mza-wtag — APPROVED** [review]

**Review by hhu-wtag — APPROVED** [review]

**Pull request closed without merging**